app/src/main/java/com/dsvag/currencyexchanger/data/models/latest/USD.kt

```diff
@@ -0,0 +1,20 @@
+package com.dsvag.currencyexchanger.data.models.latest
+
+
+import com.google.gson.annotations.SerializedName
+
+data class USD(
```

Comment:
Названия классов не принято называть капсом
```suggestion
data class Usd(
```